src/odemis/dataio/test/tiff_test.py

```diff
@@ -1506,9 +1492,6 @@ def testAcquisitionDataTIFFSmallFile(self):
         self.assertEqual(num_rows, len(tile))
         self.assertEqual(num_cols, len(tile[0]))
 
-        del rdata
```

Comment:
Please, don't remove the `os.remove()`. It's better to have the test cases clean-up after themselves, and in Windows, it's a check that the class was dereferenced and the file closed.